src/components/Search.js

```diff
@@ -17,47 +21,70 @@ class Search extends React.Component {
 
   render() {
     const b = (fn, ...args) => fn.bind(this, ...args);
-
+    const { onInitRoutePlannerWithStart, onInitRoutePlannerWithFinish, selectedResult } = this.props;
     return (
-      <Navbar.Form pullLeft>
-        <AsyncTypeahead
-          labelKey="label"
-          useCache={false}
-          minLength={3}
-          delay={500}
-          ignoreDiacritics
-          onSearch={this.props.onDoSearch}
-          options={this.props.results}
-          searchText="Hľadám…"
-          placeholder="Brusno"
-          clearButton
-          onChange={b(this.onSelectionChange)}
-          emptyLabel={'Nenašli sa žiadne výsledky'}
-          renderMenuItemChildren={(result) => (
-            <div key={result.label + result.id}
-              onMouseEnter={b(this.onSuggestionHighlightChange, result)}
-              onMouseLeave={b(this.onSuggestionHighlightChange, null)}
-            >
-              <span>{result.tags.name} </span><br/>
-              <span>({result.geojson.type}, {result.tags.type})</span>
-            </div>
-          )}
-        />
-      </Navbar.Form>
+      <div>
+        <Navbar.Form pullLeft>
+          <AsyncTypeahead
+            labelKey="label"
+            useCache={false}
+            minLength={3}
+            delay={500}
+            ignoreDiacritics
+            onSearch={this.props.onDoSearch}
+            options={this.props.results}
+            searchText="Hľadám…"
+            placeholder="Brusno"
+            clearButton
+            onChange={b(this.onSelectionChange)}
+            emptyLabel={'Nenašli sa žiadne výsledky'}
+            renderMenuItemChildren={(result) => (
+              <div key={result.label + result.id}
+                onMouseEnter={b(this.onSuggestionHighlightChange, result)}
+                onMouseLeave={b(this.onSuggestionHighlightChange, null)}
+              >
+                <span>{result.tags.name} </span><br/>
+                <span>({result.geojson.type}, {result.tags.type})</span>
+              </div>
+            )}
+          />
+        </Navbar.Form>
+        {this.props.tool === 'search' && 
+          <Navbar.Form pullLeft>
+            <ButtonGroup>
+              <Button onClick={b(onInitRoutePlannerWithStart, selectedResult)}>
+                <Glyphicon glyph="triangle-right" style={{ color: '#32CD32' }}/> Navigovať odtiaľto
```

Comment:
no, az to pouzijem treti raz, tak to vyjmem do class. ale bude to musiet ist do main.css (lebo to pouziva aj Search aj RoutePlanner), co sa mi nepaci lebo je to prilis globalne.